builders/interact/substrate-api/chopsticks.md

```diff
@@ -166,28 +164,68 @@ Chopstick's internal WebSocket server has special endpoints that allow the manip
-    - **`hashOrNumber` number | string** - if found, the chain head will be set to the block with the block number or block hash of this value  
+??? function "**dev_newBlock** (options) — Generates one or more new blocks"
+
+    === "Parameters"
+
+        - **options** - `{ "to": number, "count": number }` - a JSON object where `"to"` will create blocks up to a certain value, and `"count"` will increase by a certain number of blocks. Use only one entry at a time within the JSON object
+
+    === "Example"
+
+        ```js
+        import { WsProvider } from '@polkadot/api'
+        const ws = new WsProvider(`ws://localhost:8000`)
+        // Creates five new blocks
+        await ws.send('dev_newBlock', [{ count: 5 }])
+        ```
+
+??? function "**dev_setStorage** (values, blockHash) — Creates or overwrites the value of any storage"
+
+    === "Parameters"
+
+         - **values** - Object - a JSON object resembling the path to a storage value, similar to what you would retrieve via Polkadot.js  
+        - **blockHash** - String - optional, the block hash at which the storage value is changed  
+        
+    === "Example"
+
+        ```js
+        import { WsProvider } from '@polkadot/api';
+        const ws = new WsProvider(`ws://localhost:8000`);
+        // Overwrites the sudo key
+        await ws.send('dev_setStorage', 
+            [{"Sudo": { "Key": "0x6Be02d1d3665660d22FF9624b7BE0551ee1Ac91b" }}]
+        );
+        ```
+
+??? function "**dev_timeTravel** (date) — Sets the timestamp of the block to the date value"
+
+    === "Parameters"
+
+         - **date** - Date - a string compatible with the JavaScript Date library that will change the timestamp at which the next blocks being created will be. All future blocks will be sequentially created after that point in time  
+
+    === "Example"
+
+        ```js
+        import { WsProvider } from '@polkadot/api';
+        const ws = new WsProvider(`ws://localhost:8000`);
+        // Sets the timestamp of the block to 15th August 2030
+        await ws.send('dev_timeTravel', ["2030-08-15T00:00:00"]);
+        ```
+
+??? function "**dev_setHead** (hashOrNumber) — Sets the head of the blockchain to a specific hash or number"
+
+    === "Parameters"
+
+         - **hashOrNumber** - number | string - if found, the chain head will be set to the block with the block number or block hash of this value
+        
+    === "Example"
+
+        ```js
+        import { WsProvider } from '@polkadot/api';
+        const ws = new WsProvider(`ws://localhost:8000`);
+        // Sets the head to block number 500
+        await ws.send('dev_setHead', [500]);
+        ```
 
 Each method can be invoked by connecting to the WebSocket (`ws://localhost:8000` by default) and sending the data and parameters in the following format. Replace `METHOD_NAME` with the method's name, and replace or delete `PARAMETER_1` and `PARAMETER_2` with the parameter data relevant to the method:  
```

Comment:
I don't think this last paragraph is necessary anymore since we provide examples, but also doesn't hurt. wdyt?

Reply:
I thought that this json format would be useful to use with curl, but chopsticks doesn't open HTTP ports to interact with, so, If it's not adding any value, I think it's better to remove the last paragraph!